### PR TITLE
2382/berkeley way customizations

### PR DESCRIPTION
**Pull request by YazeedLoonat**
## Issue Overview

This PR addresses [#issue](https://github.com/bloom-housing/bloom/issues/2382)

- [x] This change addresses the issue in full

## Description
This adds a program and preference for berkeley way 

## How Can This Be Tested/Reviewed?
run the following postgres
```
INSERT INTO programs (Title, subtitle, description, form_metadata)
SELECT 
    'Flat Rent & Rent Based on Income' as Title,
    'This property includes two types of affordable housing programs. You can choose to apply for one or both programs. Each program will have its own applicant list. Your choice will tell us which list(s) to put your name on. Additional information on each of the two types of housing opportunities are below.' as subtitle,
    'Do you want to apply for apartments with flat rent and rent based on income?' as description,
    '{"key":"rentBasedOnIncome","type":"checkbox","options":[{"key":"flatRent","extraData":[],"description":true},{"key":"30Percent","extraData":[],"description":true}]}' as form_metadata
;

SELECT 
    *
FROM programs
WHERE Title = 'Flat Rent & Rent Based on Income'
;

SELECT
    *
FROM jurisdictions
WHERE name = 'Alameda'
;

INSERT INTO jurisdictions_programs_programs
SELECT '35416f6e-d580-48e9-9ed6-e7ce788a62f1' as jurisdictions_id,
'5f7b5ace-e6ba-4a24-9a55-7d0a12f89f5b' as programs_id
;

```

then add that program and preference to a listing then test it out
## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have reviewed the changes in a desktop view
- [ ] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] I have exported any new pieces added to ui-components
- [ ] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

**Comment by YazeedLoonat** [comment]
@emilyjablonski good call, got that updated

**Review by seanmalbert — CHANGES_REQUESTED** [review]
@YazeedLoonat ,

Can you make the type enumerable with "checkbox" and "radio" as the options. Otherwise, this looks good to me.

**Comment by YazeedLoonat** [comment]
Hey @seanmalbert got this updated, lemme know what you think

**Review by emilyjablonski — CHANGES_REQUESTED** [review]
Adding the seed, commented above

**Comment by YazeedLoonat** [comment]
@emilyjablonski added!

**Comment by emilyjablonski** [comment]
@YazeedLoonat You'll need to update the Cypress test

File: shared-helpers/src/programHelpers.ts
Comment (seanmalbert) [inline shared-helpers/src/programHelpers.ts:17]: Please update this to use the new enum, e.i `program?.formMetadata?.type === FormMetaDataType.checkbox`

File: backend/core/types/src/backend-swagger.ts
Comment (seanmalbert) [inline backend/core/types/src/backend-swagger.ts:4486]: I think you need to run generate:client since the type update.

File: sites/public/pages/applications/household/programs.tsx
Comment (seanmalbert) [inline sites/public/pages/applications/household/programs.tsx:127]: Like above, please update to use the enum.

**Review by seanmalbert — CHANGES_REQUESTED** [review]

**Comment by emilyjablonski** [comment]
Screen Shot 2022-02-02 at 11 09 10 AM
I can select both even though it says "select one" is that expected?

**Comment by emilyjablonski** [comment]
Screen Shot 2022-02-02 at 11 12 14 AM
Additionally if I try and change the value of the entry in the partners application edit page I get this error

**Comment by emilyjablonski** [comment]
Screen Shot 2022-02-02 at 11 14 07 AM
I also find the CSV export columns to be confusing as to what is selected

**Review by seanmalbert — APPROVED** [review]
@YazeedLoonat ,

LGTM, aside from what @emilyjablonski called out. Once that's covered and she approves it, I think it's good to merge and move over to QA.

**Review by emilyjablonski — APPROVED** [review]
LGTM once the partners netlify job passes :)

**Comment by seanmalbert** [comment]
For some reason, github is showing the tests are still running, but circleci is showing them as passed, so we're good to merge.

**Pull request merged**
